media/css/sandstone/sandstone-resp.less

```diff
@@ -166,7 +166,48 @@ a.more {
             }
         }
     }
+}
+
+// mozID visual style
+// full width rows w/differing bg colors
+// TODO: needs responsive styles
+.mozID {
+    a:link,
+    a:active,
+    a:focus,
+    a:visited {
+        color: #fff;
+
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+
+    h1, h2, h3, h4, h5, h6, .huge, .large {
+        color: #fff;
+        text-shadow: none;
+    }
+
+    #wrapper {
+        width: 100%;
+        padding-top: 64px; // make room for #masthead
+
+        .container {
+            width: (@widthDesktop - (@baseLine * 2));
+            padding-left: @baseLine;
+            padding-right: @baseLine;
+        }
+    }
 
+    #masthead {
+        position: absolute;
```

Comment:
Ah, this was from when `#wrapper` needed background styling to sit behind tabzilla. Approach changed (aurora lines now applied to `#outer-wrapper`), so can get rid of absolute positioning here.